Sources/SwiftDocC/Indexing/Navigator/NavigatorIndex.swift

```diff
@@ -136,48 +136,129 @@ public class NavigatorIndex {
     }()
     
     /**
-     Initialize an `NavigatorIndex` from a given path.
-     
-     - Parameter url: The URL pointing to the path from which the index should be read.
-     - Parameter bundleIdentifier: The name of the bundle the index is referring to.
-     - Parameter readNavigatorTree: Indicates if the init needs to read the navigator tree from the disk, if false, then `readNavigatorTree` needs to be called later. Default: `true`.
-     - Parameter presentationIdentifier: Indicates if the index has an indentifier useful for presentation contexts.
-     
-     - Throws: A `NavigatorIndex.Error` describing the nature of the problem.
+    Initialize an `NavigatorIndex` from a given path.
+    
+    - Parameters:
+        - url: The URL pointing to the path from which the index should be read.
+        - bundleIdentifier: The name of the bundle the index is referring to.
+        - readNavigatorTree: Indicates if the init needs to read the navigator tree from the disk, if false, then `readNavigatorTree` needs to be called later. Default: `true`.
+        - presentationIdentifier: Indicates if the index has an indentifier useful for presentation contexts.
+        - postNodeAction: An action to perform after reading a node.
+    
+    - Throws: A `NavigatorIndex.Error` describing the nature of the problem.
      
-     - Note: The index powered by LMDB opens in `readOnly` mode to avoid performing a filesystem lock which fails without writing permissions. As this initializer opens a built index, write permission is not expected.
-     */
-    public init(url: URL, bundleIdentifier: String? = nil, readNavigatorTree: Bool = true, presentationIdentifier: String? = nil) throws {
```

Comment:
This change might be a good opportunity to refine the docs at https://apple.github.io/swift-docc/documentation/swiftdocc/navigatorindex/